rover/src/owr_usb_reset/scripts/usb_reset.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# license removed for brevity
+import rospy
+import subprocess
+from std_msgs.msg import String,Int16
+from owr_messages.msg import devices
+
+"""in visudo, need to add %ros ALL = NOPASSWD: /home/ros/owr_software/rover/src/owr_usb_reset/usb_reset
+after %sudo	ALL=(ALL:ALL) ALL. This allows the usb reset program to run without needing a password for 
+sudo. Eneter password manually while running causes a problem 
```

Comment:
can you check there is not a group that gives this access by default?
If not can you put this setup into a bash script so its easier to setup (put it in the rover/system folder)